README.md

```diff
@@ -26,8 +26,12 @@ var params = {
   keypairId: process.env.PUBLIC_KEY,
   privateKeyString: process.env.PRIVATE_KEY,
   privateKeyPath: '/path/to/private/key',      // Optional. Use as an alternative to privateKeyString.
-  expireTime: '<epoch time when you wish the link to expire>'
+  expireTime: new Date(new Date().getTime() + (1000 * 100)) // Now + 100 seconds
```

Comment:
In df26f4e7af130c8dd7de73f78d513d8a55fb1105, I have simplified the readme to use a simple date constructor. I realize it's totally a matter of opinion - but I think the confusion for us was more to do with the conversion to unix time and how that should be done - which is why I added support for js Date. If you'd rather revert all that I certainly understand.